sourcecode/src-dotty/sourcecode/Macros.scala

```diff
@@ -120,7 +120,7 @@ object Macros {
     import qctx.tasty.{_, given}
     val owner = rootContext.owner
     val fullName = owner.fullName.trim
-      .split("\\.", -1)
+      .split("\\.", -1).map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.